src/cli.js

```diff
@@ -241,16 +242,16 @@ async function runCmd (argv, stdout, stderr) {
 
         outDir = outDir || resolve("dist");
         mkdirp.sync(outDir);
-        // remove all existing ".js" files in the out directory
+        // remove all existing ".js", ".mjs" and ".cjs" files in the out directory
         await Promise.all(
           (await new Promise((resolve, reject) =>
-            glob(outDir + '/**/*.js', (err, files) => err ? reject(err) : resolve(files))
+            glob(outDir + '/**/*.(js|mjs|cjs)', (err, files) => err ? reject(err) : resolve(files))
```

Comment:
Should `mjs` be removed here?

Reply:
Sure, until we add support for mjs that makes sense. Added.